src/components/TodoItem.js

```diff
@@ -3,22 +3,62 @@ import PropTypes from 'prop-types';
 import styles from './TodoItem.module.css';
 
 class TodoItem extends React.PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      editing: false,
+    };
+  }
+
+  handleEditing = () => {
+    this.setState({
+      editing: true,
+    });
+  }
+
+  handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      this.setState({ editing: false });
+    }
+  }
+
   render() {
+    const viewMode = {};
+    const editMode = {};
+    const { editing } = this.state;
+    if (editing) {
+      viewMode.display = 'none';
+    } else {
+      editMode.display = 'none';
+    }
     const completedStyle = {
       fontStyle: 'italic',
       color: '#595959',
       opacity: 0.4,
       textDecoration: 'line-through',
     };
```

Comment:
You are using modules and inline styled-components, I think it should be ether and not both. I am not sure about this one.

Reply:
Thank you for the review. I have fixed it